tests/Api/Response/ResponseTest.php

```diff
@@ -122,10 +122,10 @@ public function getQueryInfoMessageProvider(): array
                     'selectedVendorName' => null
                 ],
                 [],
-                'Findologic::Template.correctedQuery',
+                'Findologic::Template.queryInfoMessageQuery',
                 [
-                    'alternativeQuery' => 'Test',
-                    'originalQuery' => 'OriginalTest'
+                    'query' => 'Test',
+                    'hits' => 0
```

Comment:
Is there no new test for the changed behavior?
1. The SmartDidYouMeanCotainer now calls `->getSmartDidYouMean()`
1. Two lines of output instead of one.
1. One sdym case was broken, as stated in the story: "Fix the did you mean case."